diff --git a/src/main/java/Main.java b/src/main/java/Main.java
index be02fb2..332bd20 100644

Authors: Mikołaj Noga, Szymon Jakóbiak
CERT violations : ERR02-J, MSC17-C, MSC13-C
The problems are described in detail in CERTViolations.pdf

--- a/src/main/java/Main.java
+++ b/src/main/java/Main.java
@@ -12,7 +12,7 @@ public class Main {
     public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
         scanner.useLocale(Locale.US);
-
+        Logger logger = Logger.getLogger(String.valueOf(Main.class));
         boolean flag = true;
         while(flag){
             System.out.println("Choice number: 0.Exit, 1.(01_04), 2.(02_02) " +
@@ -75,7 +75,7 @@ public class Main {
                             System.out.println(exercise_01_06(MTBF, time));
                         } else
                             System.out.println(exercise_01_06());
-
+                        break;
                     case 5:
                     if (askForWillingnessToProvideVariables(scanner)) {
                             System.out.println("(double reliability, int time)\n\n" +
@@ -92,10 +92,10 @@ public class Main {
                         System.out.println("Wrong number");
                 }
             } catch (InputMismatchException e){
-                System.out.println("You provided wrong type of variable");
+                logger.log(Level.INFO, "You provided wrong type of variable");
                 scanner.next();
             } catch (ValueOutOfTheRangeException e){
-                System.out.println(e.getMessage());
+                logger.log(Level.INFO, e.getMessage());
             }
         }
     }
@@ -310,11 +310,4 @@ public class Main {
         return "Failure rate: " + failureRate + " , MTBF: " + MTBF;
     }
 
-    public void testMethod(){
-        int x = 13;
-        int y = x + 13;
-        y += x;
-        return;
-    }
-
 }
